docs/advanced-topics/openshift.asciidoc

```diff
@@ -16,7 +16,7 @@ This page shows how to run ECK on OpenShift.
 * <<{p}-openshift-apm,Deploy an APM Server instance with a route>>
 * <<{p}-openshift-beats,Grant privileged permissions to Beats>>
 
-NOTE: Only Elasticsearch and Kibana are compatible with the `restricted` https://docs.openshift.com/container-platform/4.1/authentication/managing-security-context-constraints.html[Security Context Constraint]. To run the APM Server on OpenShift you must allow the Pod to run with the `anyuid` SCC as described in <<{p}-openshift-apm>>.
+WARNING: With Elastic Stack 7.9, note that APM Server is now compatible with OpenShift, see <<{p}-openshift-apm>>. Enterprise Search 7.9 is not yet compatible with Openshift.
```

Comment:
> Enterprise Search 7.9 is not yet compatible with Openshift

Enterprise Search before 7.9 was compatible with the `restricted` SCC . We have an incompatibility starting with 7.9.x  On a second thought I'm wondering if the same workaround used for APM < 7.9 can be used for Enterprise Search = 7.9.x.

I need to do the test, will update my review with my findings.

Reply:
I just did the test and I can confirm that if a user really want to deploy Enterprise Search 7.9.x on Openshift then the same workaround as the one described for the APM Server < 7.9 can be applied.

The only thing I'm not sure about is how to reflect those 2 two particular cases and the workaround in the doc. I guess that if we want to broaden the scope of the workaround we have to:
* Update the title `Deploy an APM Server instance with a route` to something more generic like `Deploy Docker images with the anyuid SCC` 
* Highlight that the workaround only aims to circumvent incompatibilities with APM < 7.9 and Enterprise Search = 7.9.x

Reply:
OK, thank you @barkbay!

Reply:
@barkbay I reworked the Warning and turned the APM title to something more generic (as you suggested). Let me know if this looks any better.